landoapi/transplants.py

```diff
@@ -70,12 +70,13 @@ class LandingAssessmentState:
     """Encapsulates the state of a landing request for assessment.
```

Comment:
```suggestion
    """Encapsulates the state of a stack for assessment.
```

Reply:
This is the landing assessment state, not the stack assessment.

Reply:
Makes sense to me. :+1: